src/components/FolderList.jsx

```diff
@@ -1,16 +1,16 @@
 import { TOTAL_LIST_NAME } from '../constants/folder';
 
-const FolderList = ({ folderList, totalHandler, listHandler }) => {
+const FolderList = ({ folderList, onTotalListClick, onListClick }) => {
   if (!folderList.length) {
```

Comment:
조건을 이렇게 명시적으로 작성해주시면 더 좋을 것 같네요!
```suggestion
  if (folderList.length === 0) {
```